regression/goto-analyzer/pointer-comparison-inequality-different-target/main.c

```diff
@@ -0,0 +1,20 @@
+#include <assert.h>
+
+int main()
+{
+  int a[2];
+  int b[2];
+
+  int *p = &(a[0]);
+  int *q = &(b[1]);
+
```

Comment:
Can we have a comment in either in the test case or in the `.desc` files noting that this is undefined behaviour and so `UNKNOWN` is the expected behaviour.